lib/cluster_ht_item.rb

```diff
@@ -42,7 +43,17 @@ def delete
 
   # Deletes an HTItem, then re-adds it
   def update
-    delete
+    Cluster.where("ht_items.item_id": @ht_item.item_id).each do |c|
```

Comment:
Do we have something that makes an index for this?

Reply:
class Cluster
  ...
  index({ "ht_items.item_id": 1 }, unique: true, sparse: true)
  ...